src/components/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Grid, Paper, Container } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
```

Comment:
Since we are importing all the Material ui components in line 3, just move Divider in there as well. Theres some benefits and cons (like speed) to importing components two different ways but it wont matter much here.